supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesBackendLifecycleActorFactory.scala

```diff
@@ -82,3 +58,29 @@ case class JesBackendLifecycleActorFactory(configurationDescriptor: BackendConfi
 
   override lazy val fileHashingFunction: Option[FileHashingFunction] = Option(FileHashingFunction(JesBackendFileHashing.getCrc32c))
 }
+
+object JesBackendLifecycleActorFactory {
```

Comment:
I don't have a strong preference either way, but IMHO we should sort out a standard as a team so we don't have huge diffs thrashing the placement of companion objects to and fro.